algebird-test/src/test/scala/com/twitter/algebird/BloomFilterTest.scala

```diff
@@ -255,15 +255,13 @@ class BloomFilterTest extends WordSpec with Matchers {
         stream.toByteArray
       }
 
-      val items = (1 until 10).map { _.toString }
+      val items = (1 until 10).map(_.toString)
       val bf = BloomFilter(10, 0.1).create(items: _*)
-      val bytesBeforeSizeCalled = new String(serialize(bf))
+      val bytesBeforeSizeCalled = Bytes(serialize(bf))
       bf.size
-
       assert(bf.contains("1").isTrue)
-
-      val bytesAfterSizeCalled = new String(serialize(bf))
-      assert(bytesBeforeSizeCalled == bytesAfterSizeCalled)
+      val bytesAfterSizeCalled = Bytes(serialize(bf))
+      assert(bytesBeforeSizeCalled.size == bytesAfterSizeCalled.size)
```

Comment:
It instantiates the dense representation in the lazy Val. I wonder if some but gets flipped? If so seems like a regression in 2.12, but one that doesn't affect the behavior. Maybe we should make a minimal test case to see if this is true for all classes with a transient lazyval.